### PR TITLE
[Toggle PSR-11 Factories] Add static version to RemoveStrategyFactory

### LINKED ISSUE #135: [toggle-crud-psr11-factories] Add static version to RemoveStrategyFactory
**Epic:** https://github.com/pheature-flags/pheature-flags/issues/122 Toggle PSR-11 Factories
**Package:** https://github.com/pheature-flags/toggle-psr11-factories 

**Description:**

Following the example defined in the issue https://github.com/pheature-flags/pheature-flags/issues/122, create a static version of the RemoveStrategyFactory class.

### PR DESCRIPTION
**Pull request by xserrat**
My first contribution to the repo :) 

It closes #135

### CODE REVIEW

**Comment by xserrat** [comment]
🚀  @kpicaza! Maybe we could add an issue to add a new workflow in order to the same labels as the issue and also the assignee. What do you think? -> Moved to a [discussion](https://github.com/pheature-flags/pheature-flags/discussions/152)

**Comment by kpicaza** [comment]
Thanks @xserrat :rocket::rocket::rocket: let check the PR https://github.com/pheature-flags/pheature-flags/pull/136/files, we are adding a unit test to go increasing the coverage.

**Comment by xserrat** [comment]
> Thanks @xserrat 🚀🚀🚀 let check the PR https://github.com/pheature-flags/pheature-flags/pull/136/files, we are adding a unit test to go increasing the coverage.

Oh! Totally, I'm going to add it right now! I've fixed the construction, I missed the static 😅

**Comment by xserrat** [comment]
Done @kpicaza ! I've just added the unit tests!

File: packages/toggle-crud-psr11-factories/src/RemoveStrategyFactory.php [hunk 1]
```diff
@@ -12,9 +12,13 @@ final class RemoveStrategyFactory
 {
     public function __invoke(ContainerInterface $container): RemoveStrategy
     {
-        /** @var FeatureRepository $featureRepository */
-        $featureRepository = $container->get(FeatureRepository::class);
+        return self::create(
```
Comment (kpicaza) [inline packages/toggle-crud-psr11-factories/src/RemoveStrategyFactory.php:17]: Psalm doesn't like getting instances directly from the container

```suggestion
        /** @var FeatureRepository $featureRepository
        $featureRepository = $container->get(FeatureRepository::class);
        
        return self::create($featureRepository);
```

Comment (xserrat) [reply packages/toggle-crud-psr11-factories/src/RemoveStrategyFactory.php:17]: > Psalm doesn't like getting instances directly from the container
> 
> 
> 
> ```suggestion
> 
>         /** @var FeatureRepository $featureRepository
> 
>         $featureRepository = $container->get(FeatureRepository::class);
> 
>         
> 
>         return self::create($featureRepository);
> 
> ```

I didn't know that, I'll setup my local env again to run Psalm

**Review by kpicaza — APPROVED** [review]
:ship::ship::ship:

**Pull request merged**